Source/WebCore/Modules/WebGPU/GPUQueue.cpp

```diff
@@ -110,6 +114,18 @@ static ImageBuffer* imageBufferForSource(const auto& source)
 {
     return WTF::switchOn(source, [](const RefPtr<ImageBitmap>& imageBitmap) {
         return imageBitmap->buffer();
+    }, [](const RefPtr<ImageData>) -> ImageBuffer* {
+        ASSERT(0 && "not implemented");
```

Comment:
Will create a FIXME and add this https://bugs.webkit.org/show_bug.cgi?id=261567 bug link here